balancer-js/src/modules/data/pool/subgraphOnChain.ts

```diff
@@ -42,6 +42,7 @@ export class PoolsSubgraphOnChainRepository
 
   private filterPools(pools: Pool[]): Pool[] {
     const filteredPools = pools.filter((p) => {
+      if (p.swapEnabled === false) return false;
```

Comment:
@johngrantuk - this line prevents TestPoolHelper from working as expected when testing against pools in Recovery Mode. Is it ok if I remove it? Or should I find a way around it for these kinds of tests?